### ISSUE TITLE
sympy.limit yields wrong limit in sigmoidal expression

### ISSUE THREAD

**Issue by benmaier** [sympy/sympy#24386]
Not sure what's going wrong here, but the limit of a sigmoidal expression seems to go to zero when you add a constant, while it should not (erroneous output for `In[7]`, expected output shown in `In[8]`). Help much appreciated 

```python
In [1]: import sympy as sy

In [2]: sy.__version__
Out[2]: '1.11.1'

In [3]: A, B, t = sy.symbols("A B t")

In [4]: s = sy.Symbol("sigma", positive=True)

In [5]: expr = (A+B)/(1+sy.exp(t)); sy.pprint(expr)
A + B
──────
 t
ℯ  + 1

In [6]: sy.limit(expr,t,-sy.oo) 
Out[6]: A + B # this is correct

In [7]: sy.limit(expr-B,t,-sy.oo) 
Out[7]: -B # this is incorrect

In [8]: sy.limit(expr,t,-sy.oo) - B 
Out[8]: A # this is what the previous call should've given
```

**Comment by victorchrollo14** [comment]
I would like to work on this issue. Can anyone tell me, how to approach this.

**Comment by benmaier** [comment]
bump

**Comment by benmaier** [comment]
bump 2

**Comment by 1e9abhi1e10** [comment]
This issue is resolved in `1.12` ,  a test can be added to close the issue.

```python
>>> import sympy as sy
>>> sy.__version__
'1.12'
>>> A, B, t = sy.symbols("A B t")
>>> s = sy.Symbol("sigma", positive=True)
>>> expr = (A+B)/(1+sy.exp(t)); sy.pprint(expr)
A + B
──────
 t
ℯ  + 1
>>> sy.limit(expr-B,t,-sy.oo)
A
>>> sy.limit(expr,t,-sy.oo) - B
A
```

**Comment by benmaier** [comment]
works now! Thanks everyone!
